src/SDKs/HDInsight/Management.HDInsight/Management.HDInsight.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.HDInsight</PackageId>
+    <Description>Azure HDInsight Management SDK Library</Description>
+    <AssemblyName>Microsoft.Azure.Management.HDInsight</AssemblyName>
+    <VersionPrefix>3.1.0-preview</VersionPrefix>
```

Comment:
@wawon-msft the last nuget that is released on nuget.org is 2.0.7, why jump to 3.1.0? Why not 3.0.0?
Any particular reason to make it 3.1.0?

Reply:
That was a mistake on my part. I bumped it up from 3.0.0, but did not realize 3.0.0 wasn't released yet.